development/docker-compose.requirements.yml

```diff
@@ -2,7 +2,7 @@
 version: "3.8"
 services:
   postgres:
-    image: "postgres:14-alpine"
+    image: "postgres:13-alpine"
```

Comment:
why?

Reply:
This was done to support the Apple ARM processors. Trying to use 14 gave some SCRAM authentication error that I couldn't resolve. I figured since this only affects the development environment for the project it shouldn't be a big deal to use 13 instead of 14. It's not like we're using 14 for anything in particular.